notifications/smart_reminder_views.py

```
@@ -0,0 +1,565 @@
+        reminder = self.get_object()
+        
+        schedules = ReminderSchedule.objects.filter(
+            reminder=reminder,
+            scheduled_time__gt=timezone.now(),
+            is_sent=False
+        ).order_by('scheduled_time')[:10]
+        
+        serializer = ReminderScheduleSerializer(schedules, many=True)
+        return Response(serializer.data)
+    
+    def _has_patient_access(self, user, patient_id):
```

Ruff (0.12.2)

Missing return type annotation for private function `_has_patient_access`

(ANN202)